resources/views/mship/management/dashboard.blade.php

```diff
@@ -367,7 +367,7 @@ class="btn btn-xs btn-danger">
                     <div class="row">
                         <div class="col-xs-12">
                             @if($_account->discord_id)
-                                Currently registered with Discord ID {{ $_account->discord_id }}. <br/>
+                                Currently registered with Discord ID {{ $_account->discord_user->username ?? $_account->discord_id }}#{{ $_account->discord_user->discriminator ?? ' ' }}. <br/>
```

Comment:
A check on the `discriminator` would likely be more appropriate as the presence of a `username` attribute has no bearing on the output.

Reply:
Is this ok? https://github.com/VATSIM-UK/core/pull/2029/commits/b1e8d21b6044e905acc4d895361d1ff57f61a7bf